src/dmd/common/audit_log.d

```diff
@@ -0,0 +1,81 @@
+/**
+ * Generate text files for assistance in debugging dmd.
+ *
+ */
+
+module dmd.common.audit_log;
+import core.stdc.stdlib;
+import core.stdc.stdio;
+import dmd.root.rootobject;
+import dmd.globals;
+import dmd.ast_node;
+import dmd.hdrgen;
+import dmd.expression;
+import dmd.dsymbol;
+import dmd.statement;
+import dmd.mtype;
+import dmd.init;
+import dmd.dtemplate;
```

Comment:
All these imports violate the package this module is located in.  This can't be considered "common" so long as you have dependencies on the dmd front-end.

Reply:
Moved the file to the frontend as discussed in discord.